docs/install.rst

```diff
@@ -393,60 +393,105 @@ after changing settings you can run the following command::
-and lookup that name in the environment. For example: ``switch('the-dude')`` would look for the
-environment variable ``SWITCH_THE_DUDE``. If the value of that variable is any of "on", "true", "1", or
-"yes", then it will be considered "on", otherwise it will be "off".
-
-You can also supply a list of locale codes that will be the only ones for which the switch is active.
-If the page is viewed in any other locale the switch will always return ``False``, even in ``DEV``
-mode. This list can also include a "Locale Group", which is all locales with a common prefix
-(e.g. "en-US, en-GB" or "zh-CN, zh-TW"). You specify these with just the prefix. So if you
-used ``switch('the-dude', ['en', 'de'])`` in a template, the switch would be active for German and
-any English locale the site supports.
-
-You may also use these switches in Python in ``views.py`` files (though not with locale support).
-For example::
+Feature Flipping (aka Switches, or waffle switches)
+===================================================
+
+Switches are managed using django-waffle and are stored in the database. These switches control
+behavior and/or features of select pages on Bedrock, and their state (active or inactive) is based
+on an ``active`` boolean field in the database.
+
+Defining and Using Switches
+---------------------------
+
+The ``switch()`` template helper function allows you to check whether a specific switch is active. You
+pass a name to the function (using only letters, numbers, and dashes), which is automatically
+converted to uppercase and with dashes replaced by underscores for the lookup in the database. For
+example, ``switch('the-dude')`` will look for a switch named ``THE_DUDE`` in the database.
+
+Locale-Specific Switches
+------------------------
+
+You can provide a list of locale codes to limit the switch’s activation to specific locales. If the
+page is viewed in a locale not included in the list, the switch will return False. You can also use
+"Locale Groups," which apply to all locales with a common prefix (e.g., "en-US, en-GB" or "zh-CN,
+zh-TW"). To use these groups, pass the prefix. For example, switch('the-dude', ['en', 'de']) will
+activate the switch for German and any English locale supported by the site.
+
+Managing Switches
+-----------------
+
+Switches are managed through the Django Admin interface, where you can add, edit, or remove switches
+from the database directly. This interface allows for easy management of feature toggles without
+modifying environment variables or code.
+
+Example Usage in Templates
+--------------------------
+
+You can use the ``switch()`` helper function in your templates as follows:
+
+.. code-block:: html
+
+    {% if switch('the-dude') %}
+        <!-- Feature-specific HTML goes here -->
+    {% endif %}
+
+Example Usage in Python
+-----------------------
+
+You may also use switches in Python code (though locale support is unavailable in this context):
```

Comment:
Be good to note here that this only works in code either in or called by a view. It needs to be checked at page request time. If it were used e.g. in a urls.py file to activate URLs it would only check the switch when the file is first loaded and then never again until the instance is restarted.

Reply:
Added in the latest commit. Thanks!